src/pmhn/_trees/plot_all_mut_freq.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+paths = {
+    500: ('/home/laukeller/BSc Thesis/TreeMHN/Example/trees_500.csv', '/home/laukeller/BSc Thesis/pMHN/src/pmhn/_trees/trees_500.csv'),
```

Comment:
I think the paths (a) shouldn't be absolute here (as other people won't have directory `/home/laukeller/BScThesis/...`) and (b) writing data files to `src/pmhn` is (in most cases, there are rare exceptions) suboptimal: we want to keep `src` just for code.

I'd recommend a directory `data/` or `warmup/data` for this, but (probably even better) we'll discuss how to incorporate Snakemake at this stage.